geolib/models/dstability/states.py

```diff
@@ -43,12 +43,8 @@ class DStabilityStress(DStabilityObject):
     state_type: StateType = StateType.POP
 
     def _to_internal_datastructure(self) -> PersistableStress:
-        data = {
-            **{
-                snake_to_camel(name): value
-                for name, value in self.dict(exclude={"stochastic_parameter"}).items()
-            }
-        }
+        data = {**{snake_to_camel(name): value for name, value in self.dict().items()}}
```

Comment:
the key 'stochastic_parameter' is transformed into "StochasticParameter" but actually the key "PopStochasticParameter" is required to instantiate `PersistableStress`